v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/SchemaRegistryOptions.java

```diff
@@ -188,4 +189,52 @@ public interface SchemaRegistryOptions extends PipelineOptions {
   String getSchemaRegistryKeyPasswordSecretId();
 
   void setSchemaRegistryKeyPasswordSecretId(String keyPasswordSecretId);
+
+  @TemplateParameter.Text(
+      order = 11,
+      parentName = "schemaRegistryAuthenticationMode",
+      parentTriggerValues = {KafkaAuthenticationMethod.OAUTH},
+      optional = true,
+      description = "Client ID",
+      helpText =
+          "Client ID used to obtain client credentials grant from OAuth/OIDC identity provider.")
+  String getSchemaRegistryClientId();
+
+  void setSchemaRegistryClientId(String schemaRegistryClientId);
+
+  @TemplateParameter.Text(
+      order = 11,
+      parentName = "schemaRegistryAuthenticationMode",
+      parentTriggerValues = {KafkaAuthenticationMethod.OAUTH},
+      optional = true,
+      description = "Token Endpoint URL",
+      helpText =
+          "The HTTP(S)-based URL for the OAuth/OIDC identity provider which issues access tokens.")
+  String getSchemaRegistryTokenEndpointUrl();
+
+  void setSchemaRegistryTokenEndpointUrl(String schemaRegistryTokenEndpointUrl);
+
+  @TemplateParameter.Text(
+      order = 11,
+      parentName = "schemaRegistryAuthenticationMode",
+      parentTriggerValues = {KafkaAuthenticationMethod.OAUTH},
+      optional = true,
+      description = "Scope",
+      helpText =
+          "The access token scope used to obtain client credentials grant from OAuth/OIDC identity provider.")
+  String getSchemaRegistryScope();
+
+  void setSchemaRegistryScope(String schemaRegistryScope);
+
+  @TemplateParameter.Text(
+      order = 11,
+      parentName = "schemaRegistryAuthenticationMode",
+      parentTriggerValues = {KafkaAuthenticationMethod.OAUTH},
+      optional = true,
+      description = "Client Secret ID",
+      helpText =
+          "The secret ID used in secret manager to obtain client credentials grant from OAuth/OIDC identity provider.")
```

Comment:
Make this similar to the `KafkaReadOptions#getKafkaReadKeystorePasswordSecretId()` param?
1) Change description to: "The Google Cloud Secret Manager secret ID that contains the Client Secret to use to authenticate the Schema Registry client. Required if messageFormat = AVRO_CONFLUENT_WIRE_FORMAT and schemaRegistryAuthenticationMode = OAUTH".
2) Add an example (`projects/<PROJECT_ID>/secrets/<SECRET_ID>/versions/<SECRET_VERSION>`).
3) Does this have to point to a specific version or just `projects/<PROJECT_ID>/secrets/<SECRET_ID>` will work? What about `projects/<PROJECT_ID>/secrets/<SECRET_ID>/versions/latest`?

Please review other params and make names/descriptions/examples consistent between themselves and other, already existing params.

Reply:
Made some changes. Leaving this comment open until another review.

Reply:
LGTM now, thanks!